src/components/app_catalog/app_detail/InstallAppModal.js

```diff
@@ -124,6 +127,24 @@ const InstallAppModal = props => {
     reader.readAsText(files[0]);
   };
 
+  const updateSecretsYAML = files => {
+    var reader = new FileReader();
+
+    reader.onload = (function() {
```

Comment:
Can you please refactor this to not be an IIFE?

Reply:
Done. I'm really not sure what that was there like that.  It's not like I am trying to use that parameter. Kinda confused myself.